data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3437 @@
+				`	"Why is your ship named Tejolis?"`
+					goto tejolis
+				`	"Why is your ship hull so strong?"`
+					goto hull
+				`	"<question>"`
+					goto cloak
+				`	"What kind of weapons are you using?"`
+					goto weapon
+				`	"I am honoured, but I do not have any more questions."`
+					accept
+			`	"You are aware of what keystones are, right? They are not believed to give luck for no reason, after all. I cannot tell you much more about it, but rest assured this sword would be more useful than a gun for boarding. It was given to me by the council many years ago, matching my responsibility as well as my character."`
+				goto questions
+			label tejolis
+			`	"Tejolis is the goddess of apt thinking, which you could call intelligence. It is the aptitude to do good judgments, based on context. For instance, if you have time to choose a quick decision does not make sense, and a wrong decision is hard to justify. Likewise, when time is of the essence one must learn to do quick action, or be overwelmed by the actions of others, or lack of actions. I named this ship in her honor, just like I have a statue of her in my war room, because I follow her ways."`
+				goto questions
+			label hull
+			`	"As you have probably noticed, the hull is not in the standard Hai colors. It is, partly, a matter of taste for discretion but the biggest reason is that this ship was built out of the wreck of an old Korath warship. Those materials are more sturdy than normal Hai ones, which is why this ship is so resilient to hull damage."`
+				goto questions
+			label cloak
+			`	"Many years ago, when the Korath attacked us, they used a ghost ship. It would attack the convoys and then diseapear without a trace, without even jumping out, nothing. Then once the warships would get away he came back, always harassing us. The Ghost managed to take on a lot of convoys and escorts without issues, using its superior technology and strategic advantage. But it was no real ghost, everyone has a weakness. His weakness was that whilst we could not see it, it did not mean that it was not there anymore."`
+			`	"With this in mind, the brave Unfettered laid a trap for the ship. When it disappeared they stopped shooting like usual, and made him overconfident, doing confused movements, dropping part of their cargo as if they were in distress. So he reappeared He saw the other ships coming in but did not react to them in time and got close - too close - to the convoy. Once the fleet arrived, he disappeared but it was too late for him and his limited agility. One of the convoy ships near him rammed him, which costed the life of the proud captain but revealed the ship. Then it was just a matter of shooting it down. The cloaking device was damaged in the fight, or maybe it was already damaged before, knowing the ship never cloaked for long, so we could not extract it from the wreckage. Instead we built a ship based on that wreckage's materials, and the cloaking device is a part of it."`
```

Comment:
Weren't the raids before they split and really starting warring? Also they would probably have some kind of countermeasures to it, given their advancement; which the Hai definitely do not
The cloaking device you have came from the Korath too; this is a much weaker one (which I'll be able to make it so after a given PR I'll open anytime soon)

Reply:
Mechanically I want him to have a cloaking device because later on you'll have to protect him or fail the missions, and I can't guarantee the player will "sell"(loan) they cloaking device to him, also he needs it to get his detainer should you not provide him with one; it's easy to say he used his cloaking device for it

Reply:
Hai did not fight the Remnant, but they did fight the Builders, and the smaller Ka'het have built in cloaking.

His could perhaps be an artifact from that war, pulled out of a dissected Ka'het and integrated into various Warlord ships since?

I do myself also like the implication that the Korath occasionally snag cloaks from -somewhere- though, and while we don't know where just yet, the FW one is not some uniquely random occurrence.